internal/exportimport/config.go

```diff
@@ -51,6 +51,9 @@ type Config struct {
 	// Each exposure is inserted with the app_package_name / healthAuthorityID that it was published with
 	// Use this string to signal that a key came from the export-importer job.
 	ImportAPKName string `env:"IMPORT_APP_PACKAGE_NAME, default=exportimport"`
+	// If importing a file encounters an error, we will retry a number of times
+	// before stopping.
+	MaxImportRetries uint `env:"MAX_IMPORT_RETRIES, default=5"`
```

Comment:
this is different than what I had suggested in the bug.

I think a low # of retries without a backoff strategy will cause a pretty fast fail. (25 minutes w/ default terraform config - job runs every 5 minutes).

I don't think we should stop retrying that aggressively as then it falls to a human to notice and move the file back to pending.

Reply:
Ok, so if I had some backoff, would that be adequate?

Reply:
Ok, so we now do some backoff, and if the file is no longer present in the index, we mark it as failed.